### PR TITLE
Class2 rockpaperscissors

### PR DESCRIPTION
**Pull request by jwhaney**
## Checkpoint Rubric
This is the rubric that your instructor will use to grade your checkpoints. Please do not edit. 

**Checkpoint 1**
- [ ] All tests passed: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript: 30 points

**Checkpoint 2**
- [ ] The application works as it should: 40 points
- [ ] Proper use of documentation (commenting on code): 15 points
- [ ] Properly indented code: 15 points
- [ ] Demonstrated effective use of JavaScript and the DOM API: 30 points

**Checkpoint 3**
- [ ] Use of React: 25 points
- [ ] Accesses an API: 25 points
- [ ] Proper use of documentation (commenting on code): 25 points
- [ ] The application functions as it should: 25 points

### CODE REVIEW

File: 01week/rockPaperScissors.js [hunk 1]
```diff
@@ -7,11 +7,43 @@ const rl = readline.createInterface({
...
-
 function rockPaperScissors(hand1, hand2) {
-
   // Write code here
+  //change whatever argument is input by user to a lower case string
+  const hand1L = hand1.toLowerCase().trim();
+  const hand2L = hand2.toLowerCase().trim();
+
+  //create an array of acceptible answers
+  const myArray = ['rock', 'paper', 'scissors'];
```
Comment (HippErger) [inline 01week/rockPaperScissors.js]: nice way to evaluate the validity of the inputs!

File: 01week/rockPaperScissors.js [hunk 2]
```diff
@@ -7,11 +7,43 @@ const rl = readline.createInterface({
...
+  //const hand1inarray = ;
+  //const hand2inarray = );
 
+  //proceed if user input is in myArray (acceptible answers)
+  if (myArray.indexOf(hand1L) !== -1 && myArray.indexOf(hand2L) !== -1) {
+    //check to see if input is a tie
+    if (hand1L === hand2L){
+      return "It's a tie!";
+    }
+    //check hand one against the second hand
```
Comment (HippErger) [inline 01week/rockPaperScissors.js]: You're doing so well with handling inputs and evaluating, try refactoring to see if you can write a few function that each handle individual parts of the program.

File: 01week/rockPaperScissors.js (hunk 1 above)
Comment (jwhaney) [reply 01week/rockPaperScissors.js]: ## thanks!

File: 01week/rockPaperScissors.js (hunk 2 above)
Comment (jwhaney) [reply 01week/rockPaperScissors.js]: So not trying to get out of work or anything, but I noticed this rockpaperscissors assignment was graded on campus manager and I got a 100. Do you still need me to work on refactoring this program before I merge the PR or can I merge now? I'm good either way, just trying to get a feel on the assignment expectations here.

**Comment by HippErger** [comment]
Go ahead and merge! I'm just giving you more of a challenge.

**Comment by jwhaney** [comment]
Ok cool thanks. I'll go ahead and merge.

**Pull request merged**